src/cr/cube/dimension.py

```diff
@@ -614,7 +614,12 @@ def _dimension_transforms_dict(self) -> Dict:
     def _view_insertion_dicts(self) -> List[Optional[Dict]]:
         """List of insertion dicts included in the dimension view."""
         view = self._dimension_dict.get("references", {}).get("view") or {}
-        return view.get("transform", {}).get("insertions", [])
+        insertions = view.get("transform", {}).get("insertions", [])
+        view_insertions = [
```

Comment:
Can you add a comment like `If there's no id defined on the insertion defined on the variable view, use the 1-based positional index as the id to match frontend behavior`